From d0004352f731fd158d76d6140b94f02975ed2a6a Mon Sep 17 00:00:00 2001
From: Yaz Khoury <yaz.khoury@gmail.com>
Date: Wed, 7 Aug 2019 12:16:28 -0400
Subject: [PATCH] fix: Change URLs to point to ethereumclassic repo

Some were on ethereumproject, updated to point to correct repo.
---
 _specs/ecip-1000.md | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/_specs/ecip-1000.md b/_specs/ecip-1000.md
index d5d5e72..1debd53 100644
--- a/_specs/ecip-1000.md
+++ b/_specs/ecip-1000.md
@@ -44,7 +44,7 @@ Asking the Ethereum Classic community first if an idea is original helps prevent
 
 It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum Classic is used.
 
-**2:** Once the champion has asked the Ethereum Classic community as to whether an idea has any chance of acceptance, a **draft ECIP** should be submitted to the [ECIPs git repository](https://github.com/ethereumproject/ECIPs) as a pull request. This gives the author a chance to flesh out the draft ECIP to make it properly formatted, of high quality, and to address additional concerns about the proposal. This draft must be written in ECIP style as described below, and named with an alias such as "ecip-johndoe-infinitecoins" until an editor has assigned it an ECIP number (authors MUST NOT self-assign ECIP numbers).
+**2:** Once the champion has asked the Ethereum Classic community as to whether an idea has any chance of acceptance, a **draft ECIP** should be submitted to the [ECIPs git repository](https://github.com/ethereumclassic/ECIPs) as a pull request. This gives the author a chance to flesh out the draft ECIP to make it properly formatted, of high quality, and to address additional concerns about the proposal. This draft must be written in ECIP style as described below, and named with an alias such as "ecip-johndoe-infinitecoins" until an editor has assigned it an ECIP number (authors MUST NOT self-assign ECIP numbers).
 
 ECIP authors **are responsible for collecting community feedback** on both the initial idea and the ECIP before submitting it for review. However, wherever possible, long open-ended discussions on public groups or mailing lists should be avoided. Strategies to keep the discussions efficient include: setting up a separate SIG mailing list for the topic, having the ECIP author accept private comments in the early design phases, setting up a wiki page or git repository, etc. ECIP authors should use their discretion here.
 
@@ -86,7 +86,7 @@ ECIP editors are intended to fulfill administrative and editorial responsibiliti
 
 ### Steps
 
-**1:** Each new ECIP should first be submitted as a **"pull request"** to the [ECIPs git repository](https://github.com/ethereumproject/ECIPs). Then, an editor does the following:
+**1:** Each new ECIP should first be submitted as a **"pull request"** to the [ECIPs git repository](https://github.com/ethereumclassic/ECIPs). Then, an editor does the following:
 
 * Read the ECIP to check if it is ready: sound and complete. The ideas must make technical sense, even if it doesn't seem likely to be accepted.
 * The title should accurately describe the content.
@@ -251,7 +251,7 @@ Pages must be named after the full ECIP number (eg, "ECIP 0001") and placed in t
 
 For example, the link for ECIP 1 will be:
 
-	https://github.com/ethereumproject/ECIPs/wiki/Comments:ECIP-0001 .
+	https://github.com/ethereumclassic/ECIPs/wiki/Comments:ECIP-0001 .
 
 Comments posted to this wiki should use the following format:
 
@@ -272,7 +272,7 @@ Summary tones may be chosen from the following, but this ECIP does not intend to
 For example, the preamble to ECIP 1 might be updated to include the line:
 
     Comments-Summary: No comments yet.
-    Comments-URI: https://github.com/ethereumproject/ECIPs/wiki/Comments:ECIP-1000
+    Comments-URI: https://github.com/ethereumclassic/ECIPs/wiki/Comments:ECIP-1000
                   https://some-other-wiki.org/ECIP_1_Comments
 
 These fields must follow the **Discussions-To** header defined in ECIP 1 (if that header is not present, it should follow the position where it would be present; generally this is immediately above the **Status** header).
